sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java

```diff
@@ -158,7 +158,7 @@ public Page build() {
       if (name == null || name.isEmpty()) {
         throw new IllegalArgumentException("Name must be defined and not empty");
       }
-      if (qualifiers.length > 0 && GLOBAL.equals(scope)) {
+      if (qualifiers.length > 0 && !COMPONENT.equals(scope)) {
         throw new IllegalArgumentException(format("The scope must be '%s' when component qualifiers are provided", COMPONENT));
```

Comment:
The API has accessed to [guava](https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/pom.xml#L43), so you can use it.